python/foglamp/services/core/server.py

```diff
@@ -60,6 +60,34 @@ class Server:
     service_monitor = None
     """ foglamp.microservice_management.service_registry.Monitor """
 
+    _service_name = 'FogLAMP'
+    """ The name of this FogLAMP service """
+
+    _service_description = 'FogLAMP REST Services'
+    """ The description of this FogLAMP service """
+
+    _SERVICE_DEFAULT_CONFIG = {
+        'name': {
+            'description': 'The name of this FogLAMP service',
+            'type': 'string',
+            'default': 'FogLAMP'
+        },
+        'description': {
+            'description': 'The description of this FogLAMP service',
+            'type': 'string',
+            'default': 'The FogLAMP administrative API'
+        }
+    }
+
+    _MANAGEMENT_SERVICE = '_foglamp_manage._tcp'
+    """ The management service we advertise """
+
+    _ADMIN_API_SERVICE = '_foglamp_admin._tcp'
+    """ The admin REST service we advertise """
+
+    _USER_API_SERVICE = '_foglamp_user._tcp'
```

Comment:
please remove _ between foglamp and user 

when these admin + user api is a single service; then why are we announcing twice; only diff will be authentication (sooner) + role (in future)

Reply:
They have always been separate logical services. At the moment they are the same port but they may no the forever.